### ISSUE TITLE
REQUEST: Option for board without oled display

### ISSUE THREAD

**Issue by strobel1x** [kizniche/ttgo-tbeam-ttn-tracker#27]
I'm getting error messages as I have no oled connected to my TTGO T-Beam.
A config switch do disable oled display would be nice.

**Comment by kizniche** [comment]
Good idea. I thought there was one in there, but perhaps the latest round of code changes removed it. I'll look into adding it to the config.
